tools/decoder.py

```diff
@@ -143,9 +145,13 @@ def format_address(address):
             for pair in pairs:
                 name, addr = pair.split("=")
                 if name in ["epc1", "excvaddr"]:
-                    output = format_address(addr)
-                    if output:
-                        print(f"{name}={output}")
+                    addr = addr.rstrip(',')
```

Comment:
Where does the comma show up here?

Reply:
With the below input, there is `epc1=0x40206b4b,` (<- comma)
and two lines below `epc1=0x40206b4b` (<- no comma)
Both are showing the same decoded info:
```
epc1=0x40206b4b: _printf_i at /workdir/repo/newlib/newlib/libc/stdio/nano-vfprintf_i.c:189 (discriminator 1)
```

input:
```
Fatal exception 29(StoreProhibitedCause):
epc1=0x40206b4b, epc2=0x00000000, epc3=0x00000000, excvaddr=0x00000000, depc=0x00000000

To make this dump useful, DECODE IT - https://tinyurl.com/8266dcdr
--------------- CUT HERE FOR EXCEPTION DECODER ---------------

Exception (29):
epc1=0x40206b4b epc2=0x00000000 epc3=0x00000000 excvaddr=0x00000000 depc=0x00000000

>>>stack>>>
```

Reply:
Maybe ignore or silence it instead?
(same for below as it turns out)

Totally forgot this existed... quick search shows it around `user_uart_wait_tx_fifo_empty`
```
4027a381:       ffbf21                  l32r    a2, 4027a280
```

But not sure if it is really that func or part of some other stuff. Wonder if it can be turned off in favour of Core postmortem func